### DISCUSSION TITLE
Symlink Error with Adding Cookie Notice to Github Page

### DISCUSSION THREAD

**Discussion by arcadiapage** [community/community#23633] [Other Feature Feedback, Questions, & Ideas]
<p>Hello! I’m trying to add a cookie notice to my site. I followed the directions exactly from Jekyll Codex</p>
<p><a href="https://jekyllcodex.org/without-plugin/cookie-consent/" rel="noopener nofollow ugc">jekyllcodex.org/without-plugin/cookie-consent</a></p>
<p>I uploaded the file to the _includes folder on github, and added the needed code to the necessary layout page.</p>
<p>But I keep getting this error.</p>
<p><em>The page build failed for the <code>master</code> branch with the following error:</em></p>
<p><em>A file was included in <code>/_layouts/base.html</code> that is a symlink or does not exist in your <code>_includes</code> directory. For more information, see <a href="https://docs.github.com/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#file-is-a-symlink" rel="noopener nofollow ugc">https://docs.github.com/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#file-is-a-symlink</a>.</em></p>
<p>The issue is, the required file is already in the _includes folder. I also copy and pasted the name of the file into the code of the layout file to make sure all the spelling is right. I’m not sure what I’m missing or why the cookie-consent.html file is not being seen in my _includes folder.</p>
<p>Any help will be appreciated!</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/arcadiapage">@arcadiapage</a>, welcome to the GitHub Support Community!</p>
<p>Can you post a link to the repository you’re having trouble with? It’s a little hard to know what’s going on without being able to see it, so that’d be really helpful!</p>

**Comment by arcadiapage** [comment]
<p>Here’s the link to my repository: <a href="https://github.com/arcadiapage/arcadiapage.github.io" rel="noopener nofollow ugc">https://github.com/arcadiapage/arcadiapage.github.io</a></p>
<p>I found a work around by going to a site that generates a cookie notice and then adding the code for that to the head.html file. But I’d rather see if I can get things working using the suggested method, so I removed that, and recreated the problem.</p>
<p>Thanks for your help!</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Thanks for the link! It looks like this error is happening because the HTML template you’re including for your cookie consent banner has 2 includes of its own that don’t exist in your repository:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/arcadiapage/arcadiapage.github.io/blob/785576ae06e9c765bec354b3357d2ea8618ff3dd/_includes/cookie-consent.html#L35-L36" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/arcadiapage/arcadiapage.github.io/blob/785576ae06e9c765bec354b3357d2ea8618ff3dd/_includes/cookie-consent.html#L35-L36" target="_blank" rel="noopener">arcadiapage/arcadiapage.github.io/blob/785576ae06e9c765bec354b3357d2ea8618ff3dd/_includes/cookie-consent.html#L35-L36</a></h4>
<pre class="onebox"><code class="lang-html"><ol class="start lines" start="35" style="counter-reset: li-counter 34 ;">
<li>{% include ga.js %}</li>
<li>{% include chatbutton.js %}</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>You’ll either need to add these templates to your repository or remove these includes before your site will build successfully.</p>

**Comment by arcadiapage** [comment]
<p>Thank you! I removed the two includes from the HTML template, and now it works.</p>
